### PR TITLE
Added Hindi Localisation

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes issue #900 

Changes: Added Hindi Localisation

Demo Link: http://hindi-india.surge.sh/

Screenshots of the change:

### CODE REVIEW

**Comment by anshumanv** [comment]
Isn't this for issue #900 ? :smile: 
Please deploy correctly - 
image

File: src/components/ChatApp/Settings/Settings.react.js [hunk 1]
```diff
@@ -446,11 +446,14 @@ class Settings extends Component {
 				lang: 'de-DE',
 				name: 'Deutsch'
 			}, {
-               			lang: 'am-AM',
-                		name: 'Armenian'
-            		}, {
+               	lang: 'am-AM',
```
Comment (anshumanv) [inline src/components/ChatApp/Settings/Settings.react.js]: Not indented correctly, please fix. :smile:

File: CNAME [hunk 2]
```diff
@@ -0,0 +1 @@
+unruly-camp.surge.sh
```
Comment (anshumanv) [inline CNAME]: Not required.

File: src/components/ChatApp/Settings/Settings.react.js (hunk 1 above)
Comment (aashutoshrathi) [reply src/components/ChatApp/Settings/Settings.react.js]: Actually, I fixed indentation as it was not correct in PR merged just before it. 😉

**Comment by mariobehling** [comment]
Thanks!

**Pull request merged**